design_0/hyp1.py

```diff
@@ -1,26 +1,38 @@
-import math, os
+"""This module contains all the functions needed to successfully run
+find_hypotenuse.py
+"""
 
-a = input('a:')
-b = input('b:')
+import math
 
-def calc1(a,b,c,nc=False, logging=True):
-	if c:
-		return c
+def assert_and_convert(user_input):
+    """assert_and_convert(unknown) -> float
 
-	if logging: 
-		print "a is ", a
-		a = pow(a,2)
-	if logging:
-		print "b is ", b
-		a = pow(a,2)
-	c = a + b
+    Assert that the input given by the user is appropriate. Convert
+    that input into a float. Return the input.
+    """
+    assert (isinstance(user_input, float) or
+            isinstance(user_input, int)), 'Value must be of type int or type float.'
+    assert user_input > 0, 'Value should be greater than zero.'
+    if isinstance(user_input, int):
+        float(user_input)
+    return user_input
 
-	
-	c2 = math.sqrt(c)
+def get_input():
+    """get_input() -> (float, float)
 
-	if nc:
-		c = 2 * c * math.pi
-		return c
-	return c2
+    Instruct and gather user input of two sides of a right
+    triangle.
+    """
+    print 'Enter the two sides of your right triangle.'
+    input_a = input('a:')
```

Comment:
given you'll be converting at the outset to a float, you may not even need your assert and convert, you'd only need the assert > 0 after that point.